numpy/core/shape_base.py

```diff
@@ -438,30 +437,28 @@ def _block_check_depths_match(arrays, parent_index=[]):
         return parent_index, _nx.ndim(arrays)
 
 
-def _block(arrays, max_depth, result_ndim):
+def atleast_nd(a, ndim):
```

Comment:
Leading underscore - we don't want to make this public as part of an optimization